test/test_log.py

```diff
@@ -80,6 +86,32 @@ def test_handler_filtering(self):
         logger.warning('bar')
         self.assertEqual(self.backend.calls(), [('WARNING', 'bar')])
 
+    def test_debug_logging(self):
+        buffer = io.StringIO()
+        ops.log.setup_root_logging(self.backend, debug=True, debug_stream=buffer)
```

Comment:
Check `test_infra.py`, l.43, for an easy way to test if something went to stdout/stderr.

Reply:
As a general guideline I'm *much* more in favor of code and tests that do dependency injection rather than patching a global object. (You can never run the latter in parallel, and if you ever make a mistake cleaning up you break *other* tests that are hard to debug.)
I can happily pass sys.stderr from where this is called if you feel it should be exercised live.